beta/examples/tensorflow/classification/README.md

```diff
@@ -115,5 +115,5 @@ To export a model to the OpenVINO IR and run it using the Intel® Deep Learning
 |ResNet50|INT8 w:sym,per-tensor a:sym,per-tensor|ImageNet|75.04|[resnet50_imagenet_int8.json](configs/quantization/resnet50_imagenet_int8.json)|[Link](https://storage.openvinotoolkit.org/repositories/nncf/tensorflow/models/develop/resnet50_int8_w_sym_t_a_sym_t.tar.gz)|
 |ResNet50|Sparsity 50% (Magnitude)|ImageNet|75|[resnet50_imagenet_magnitude_sparsity.json](configs/sparsity/resnet50_imagenet_magnitude_sparsity.json)|[Link](https://storage.openvinotoolkit.org/repositories/nncf/tensorflow/models/develop/resnet50_sparsity_50.tar.gz)|
 |ResNet50|INT8 w:sym,per-tensor a:sym,per-tensor + Sparsity 50% (Magnitude)|ImageNet|74.46|[resnet50_imagenet_magnitude_sparsity_int8.json](configs/sparsity_quantization/resnet50_imagenet_magnitude_sparsity_int8.json)|[Link](https://storage.openvinotoolkit.org/repositories/nncf/tensorflow/models/develop/resnet50_int8_w_sym_t_a_sym_t_sparsity_50.tar.gz)|
-|ResNet50|Filter Pruning 40%|ImageNet|74.98|[resnet50_imagenet_pruning_geometric_median.json](configs/pruning/resnet50_imagenet_pruning_geometric_median.json)|[Link](https://storage.openvinotoolkit.org/repositories/nncf/tensorflow/models/develop/resnet50_pruning_40.tar.gz)|
+|ResNet50|Filter Pruning 40%, 4.29/7.75 GFLOPS, 15.8/25.5 MParams|ImageNet|74.98|[resnet50_imagenet_pruning_geometric_median.json](configs/pruning/resnet50_imagenet_pruning_geometric_median.json)|[Link](https://storage.openvinotoolkit.org/repositories/nncf/tensorflow/models/develop/resnet50_pruning_40.tar.gz)|
```

Comment:
Could you directly provide information on how the MParams and GFlops are reduced? What do you think about creating a separate table for pruned models with additional columns such as `GFlops reduction` and `MParams reduction` where you can provide a percentage of pruning and absolute numbers.